### ISSUE TITLE
Easy: Add top-level await for es2020 modules

### ISSUE THREAD

**Issue by dtruffaut** [microsoft/TypeScript#37709]
**TypeScript Version:**  
3.9-beta

**Search Terms:** 
top level await es2020 module

**tsconfig.json**
"target":"es2020"
"module":"es2020"

**Code**
```ts
await 1;
```

**Expected behavior:**

OK if "module":"esnext" in tsconfig.json
OK if "module":"es2020" in tsconfig.json

**Actual behavior:**

OK if "module":"esnext" in tsconfig.json
KO if "module":"es2020" in tsconfig.json <----- :'(

`Top-level 'await' expressions are only allowed when the 'module' option is set to 'esnext' or 'system', and the 'target' option is set to 'es2017' or higher.ts(1378)`

**Playground Link:** 

"module":"es2020" is not available on Typescript playground.
"module":"es2020" is not available in documentation.

**Related Issues:** 

Add top-level await for esnext and system modules
https://github.com/microsoft/TypeScript/pull/35813

Basically, just add "es2020" to the list.
Easy to fix, bro.

**Comment by RyanCavanaugh** [comment]
The ES2020 feature list has not been ratified yet, so we can't definitively say TLA will be part of it, but we're expecting that to happen at the upcoming TC39 meeting.

**Comment by DanielRosenwasser** [comment]
ES2020 isn't up for any new features, and top-level `await` is still stage 3.

**Comment by DanielRosenwasser** [comment]
I have misspoken slightly - Ryan is right, and that combined with the spec ratification timeline, means that some features that hit stage 4 at the current meeting might make it into ES2020 (e.g. `import.meta`). But top-level `await` isn't up for discussion at this meeting.

**Comment by mitar** [comment]
So, top-level await has been standardized. What this means for this issue?
